src/structures/GuildChannel.js

```diff
@@ -283,6 +282,9 @@ class GuildChannel extends Channel {
    * @property {boolean} [nsfw] Whether the channel is NSFW
    * @property {number} [bitrate] The bitrate of the voice channel
    * @property {number} [userLimit] The user limit of the channel
+   * @property {string} [parent] The parent ID of the channel
+   * @param {ChannelCreationOverwrites[]|Collection<Snowflake, PermissionOverwrites>} [overwrites]
```

Comment:
`@property`

Reply:
coming back to "note to self: stop copy-pasting", done